samza-core/src/main/java/org/apache/samza/container/grouper/task/GroupByContainerIds.java

```diff
@@ -89,7 +89,7 @@ public GroupByContainerIds(int count) {
     // Convert to a Set of ContainerModel
     Set<ContainerModel> containerModels = new HashSet<>();
     for (int i = 0; i < containerCount; i++) {
-      containerModels.add(new ContainerModel(containersIds.get(i), taskGroups[i]));
+      containerModels.add(new ContainerModel(containersIds.get(i), -1, taskGroups[i]));
```

Comment:
We are setting it to -1 because there is no integer equivalent of the processorId anymore. 
I believe we are using "i" before just as index to the array of processorIds passed in the argument. 
I added a comment next to the -1 to clarify. Thanks!